spec/index.html

```diff
@@ -351,21 +350,35 @@ <h3>Triple Terms and Reification</h3>
     </figure>
 
     <p>Here is a variation on the graph shown in <a href="#fig-triple-term"></a>. This represents a graph
-      where the <a>triple term</a> corresponds to an <a>asserted triple</a>.
+      where an <a>asserted triple</a> corresponds to the <a>triple term</a> object of a <a>reifying triple</a>,
+      qualifying the description about this <a>reifier</a> as a <a>triple annotation</a>.
     </p>
 
     <figure id="fig-asserted-triple-term">
       <a href="asserted-triple-term.svg">
         <object data="asserted-triple-term.svg" aria-describedby="fig-asserted-triple-term-alt"
-          aria-label="An RDF graph containing a reifying triple where the triple term corresponds to an asserted triple"></object>
+          aria-label="An RDF graph containing a triple annotation, where the triple term of a reifying triple corresponds to an asserted triple"></object>
       </a>
       <figcaption id="fig-asserted-triple-term-alt">
-        An <a>RDF graph</a> containing a <a>reifying triple</a> where the <a>triple term</a> corresponds to an <a>asserted triple</a>.
+        An <a>RDF graph</a> containing a <a>triple annotation</a>, where the <a>triple term</a> of a <a>reifying triple</a> corresponds to an <a>asserted triple</a>.
         The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
       </figcaption>
     </figure>
 
-    <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
+    <p class="note">
+      Since <a>triple terms</a> denote abstract, logical <a>propositions</a>,
+      they are only allowed as objects of other triples.
+      Assertions involving them should commonly use <a>reifying triples</a>;
+      the <a>reifier</a> is then used for further description.
```

Comment:
I'm not quite comfortable with these two sentences:
* The causality conveyed by "since" is not obvious, and even controversial (remember that we did not reach consensus on this). Therefore I would not even try to justify this constraint.
* In the 2nd sentence "assertions involving [propositions]" may give the wrong impressions that the reifier also denotes the proposition.


```suggestion
      <a>Triple terms</a> always denote abstract, logical <a>propositions</a>,
      while <a>reifiers</a> may denote a variety of things that are related to,
      or described by, these propositions
      (e.g. a statement or a belief that the proposition holds, an event or a circumstance that makes the proposition true, etc.).
      The vast majority of use-cases brought to the Working Group deal with resources of the latter category;
      it is therefore expected that triple terms are mostly (if not exclusively) used as <a>object</a> of <a>reifying triples</a>,
      and that the reifiers (rather than the triple terms) are used in further descriptions.
```

Reply:
I sympathize with this suggestion (and while I maintain my position on the topic, this should certainly be neutrally phrased). Some thoughts:

* Should the Working Group be mentioned as such here?
* Reifiers "described by [these] propositions" might perhaps convey that the propositions related to them as subjects or objects ("weak self-referencing")... While I'd _like_ to get into the details (propositions being induced, reifiers "underlying" them, etc.), I think we should merely say "related to" here (implying via `rdf:reifies`).
* I think we got to "commonly" to stay neutral about other uses, so I'd suggest we keep that (rather than "mostly (if not exclusively)", which I otherwise mostly agree with (if not exclusively...)).

I'll propose a variant based on your suggestion and these thoughts (and change the note to a `<div>` with `<p>`:s, it was already a bit dense).